README.md

```diff
@@ -147,6 +147,11 @@ buildTypes {
 }
 ```
 
+## Flutter 3.x.x issues
+Since Flutter 3.x.x was introduced, it exposed some race conditions resulting in occasional crashes upon map disposal. The parameter `useDelayedDisposal` was introduced as a workaround for this issue until Flutter and/or Mapbox fix this issue properly. Use with caution.
```

Comment:
```suggestion
Since Flutter 3.x.x was introduced, it exposed some race conditions resulting in occasional crashes upon map disposal. The parameter `useDelayedDisposal` was introduced as a workaround for this issue until Flutter and/or MapLibre fix this issue properly. Use with caution.
```